resources/puppetlabs/lein-ezbake/template/pe/ext/redhat/init.suse.erb

```diff
@@ -84,16 +75,13 @@ start() {
     # startproc will change users, so make sure that user has permission
     # to access the present working directory.
     cd "${INSTALL_DIR}"
-    startproc -u "${USER}" -l "${LOGFILE}" -p "${PIDFILE}" "/opt/puppetlabs/server/apps/${realname}/bin/${realname} start >> /var/log/puppetlabs/${realname}/${realname}-daemon.log 2>&1"
+    startproc -u "${USER}" -l "${LOGFILE}" -p "${PIDFILE}" -w -- "${INSTALL_DIR}/bin/${realname}" start >/dev/null 2>&1
```

Comment:
I changed the command line passed to `startproc` here because the original form did not work.  `startproc` would error out because it appeared to want an executalble file as the first argument after the `${PIDFILE}` one - not an argument composed of an executable and the other arguments to be passed to that executable.

The latest code on the ![stable branch](https://github.com/puppetlabs/ezbake/blob/8095ebc2118c527a61fb34dac08013fa6b274255/resources/puppetlabs/lein-ezbake/template/pe/ext/redhat/init.suse.erb#L87) formats each of those as separate arguments rather than a single one.  That approach worked fine for me in testing on SLES 11.

I hadn't been successful in getting the piped output from the start command to be appended into the daemon.log file.  It does appear that the target of the -l option, `"${LOGFILE}"`, receives output from the start command and launched Java process, though.  `${LOGFILE}` here is set to `/var/log/puppetlabs/puppetserver/puppetserver.log`.  It would seem more consistent with other non-systemd OSes if we were to use `/var/log/puppetlabs/puppetserver/puppetserver-daemon.log` instead here although it would arguably be a compatibility break for prior users of PE on SLES 11 to switch the log file target at this point.

I'd be curious to hear what others think here.  Worthwhile to switch the -l argument to the daemon.log file instead at this point?